app/src/main/java/ca/llamabagel/transpo/search/ui/SearchAdapter.kt

```diff
@@ -19,18 +19,38 @@ class SearchAdapter(private val searchResultClickListener: (SearchResult) -> Uni
         val layoutInflater = LayoutInflater.from(parent.context)
         return when (viewType) {
             SEARCH_CATEGORY_HEADER_LAYOUT -> SearchCategoryViewHolder(
-                DataBindingUtil.inflate(layoutInflater, SEARCH_CATEGORY_HEADER_LAYOUT, parent, false)
+                DataBindingUtil.inflate(
+                    layoutInflater,
+                    SEARCH_CATEGORY_HEADER_LAYOUT,
```

Comment:
Did ktlint formatter do this? It can fit on 1 line...?

Reply:
Yes, I think this was ktlint. It's a minor formatting change that I don't think will matter whichever way we format it.